/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 2afd7ae68e8b is related to another bug (695635).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 9eda39f28e5a looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 9eda39f28e5a is related to another bug (711263).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author cpearce@mozilla.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 847837 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
analyze_commits.py:81: UserWarning: Author mwoodrow@mozilla.com could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision cf209d9bc29c is related to another bug (1267524).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1127198 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 78bde999fddd is related to another bug (975782).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 824395 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 1e937319fe2b is related to another bug (682329).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author ryanvm@gmail.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 376dc3151e42 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 62eaf89ab82f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision cc20208a6eb4 is related to another bug (1034247).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Reviewer wmccloskey@mozilla.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 5e1dd28ede5d was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 631b3d5d54f4 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author ehsan@mozilla.com could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 1bce26e37922 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 2a737ee5d9b1 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Reviewer kaie@kuix.de is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b07c0925efe5 is related to another bug (595364).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision d6242b24bc47 is related to another bug (1138967).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author ryanvm@gmail.com could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: b3113b7753c1 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f58ce7ac1c7f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 2b5753e09a92 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision dafd618c3f52 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 62aa68e8b499 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 38efa8f2e56e was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e416503aea99 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b2257226899f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7a73873e133d was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 2a607cddc4cb was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision dfde9d47d8c4 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision cf9de5c367a5 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7cf7a6ea28c2 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 55dba2167b36 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 4d6a52c5fb75 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 73c2707c4f60 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 21aab02fb41c was not found.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author karlt@mozbugz.karlt.net is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
analyze_commits.py:81: UserWarning: Author wmccloskey@mozilla.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
analyze_commits.py:81: UserWarning: Reviewer edwin@mozilla.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
Loaded 35815 bugs.
Total number of bugs: 35815
1 out of 978
2 out of 978
3 out of 978
4 out of 978
5 out of 978
6 out of 978
7 out of 978
8 out of 978
9 out of 978
10 out of 978
11 out of 978
Error with bug 1125970: Too many matching authors (cpearce@mozilla.com, bobbyholley@gmail.com) found for cpearce@mozilla.com
12 out of 978
13 out of 978
14 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Matt%20Woodrow&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
15 out of 978
16 out of 978
17 out of 978
18 out of 978
19 out of 978
20 out of 978
21 out of 978
22 out of 978
23 out of 978
24 out of 978
25 out of 978
Error with bug 919838: 
26 out of 978
27 out of 978
28 out of 978
Error with bug 1248913: Too many matching authors (mstange@themasta.com, ryanvm@gmail.com) found for ryanvm@gmail.com
29 out of 978
30 out of 978
31 out of 978
32 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=matspal&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 944291: 'email'
33 out of 978
34 out of 978
35 out of 978
36 out of 978
37 out of 978
38 out of 978
39 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=behdad&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1119776: 'email'
40 out of 978
41 out of 978
42 out of 978
43 out of 978
44 out of 978
45 out of 978
46 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
47 out of 978
48 out of 978
49 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ryan%20VanderMeulen&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
50 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 709492: 'email'
51 out of 978
52 out of 978
Error with bug 1098126: Too many matching authors (karlt@mozbugz.karlt.net, ajones@mozilla.com) found for karlt+@karlt.net
53 out of 978
54 out of 978
55 out of 978
Error with bug 831626: Too many matching authors (wmccloskey@mozilla.com, dvander@alliedmods.net) found for wmccloskey@mozilla.com
56 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
57 out of 978
58 out of 978
59 out of 978
60 out of 978
61 out of 978
62 out of 978
63 out of 978
64 out of 978
65 out of 978
66 out of 978
67 out of 978
68 out of 978
69 out of 978
70 out of 978
71 out of 978
72 out of 978
73 out of 978
74 /home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f1f81ccfb6d4 is related to another bug (1279063).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 58605e9a6ea1 is related to another bug (912303).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author jwalden@mit.edu could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 350ba395c507 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 581a0260b08b looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 713643 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 03701a9c0f82 is related to another bug (932112).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision cb31ec3a3325 is related to another bug (584168).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 57acaa90d7c6 is related to another bug (1153123).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 15a8086a2431 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision a829f51aae56 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 8c521a802625 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 2cd9d6be6dc2 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e385645fb787 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 2980ed8667f3 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 8c6ea94c7a9e was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 904720 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 812341 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 72c49203f293 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 78a029898775 is related to another bug (961091).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 1b91cf5c8407 is related to another bug (949475).
  revs, backout_comments = get_commits_for_bug(bug)
out of 978
Error with bug 1234422: Too many matching reviewers (m_kato@ga2.so-net.ne.jp, masayuki@d-toybox.com) found for m
75 out of 978
76 out of 978
77 out of 978
78 out of 978
79 out of 978
80 out of 978
81 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 821577: 'email'
82 out of 978
83 out of 978
84 out of 978
85 out of 978
Error with bug 1120128: Uplift either accepted or rejected.
86 out of 978
87 out of 978
88 out of 978
89 out of 978
90 out of 978
91 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Jeff%20Walden&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
92 out of 978
93 out of 978
94 out of 978
95 out of 978
96 out of 978
97 out of 978
98 out of 978
99 out of 978
100 out of 978
101 out of 978
102 out of 978
103 out of 978
104 out of 978
105 out of 978
106 out of 978
107 out of 978
108 out of 978
109 out of 978
110 out of 978
111 out of 978
112 out of 978
113 out of 978
114 out of 978
115 out of 978
116 out of 978
117 out of 978
118 out of 978
119 out of 978
120 out of 978
121 out of 978
122 out of 978
123 out of 978
124 out of 978
125 out of 978
126 out of 978
127 out of 978
128 out of 978
129 out of 978
130 out of 978
131 out of 978
132 out of 978
133 out of 978
134 out of 978
135 out of 978
136 out of 978
137 out of 978
138 out of 978
139 out of 978
140 out of 978
141 out of 978
142 out of 978
143 out of 978
144 out of 978
145 out of 978
146 out of 978
147 out of 978
148 out of 978
149 out of 978
150 out of 978
151 out of 978
152 out of 978
153 out of 978
154 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=woof&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1022773: 'email'
155 out of 978
Error with bug 1098126: Too many matching authors (karlt@mozbugz.karlt.net, ajones@mozilla.com) found for karlt+@karlt.net
156 out of 978
157 out of 978
158 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ryan%20VanderMeulen&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
159 out of 978
160 out of 978
161 out of 978
162 out of 978
163 out of 978
164 out of 978
165 out of 978
166 out of 978
167 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
168 out of 978
169 out of 978
170 out of 978
171 out of 978
172 out of 978
173 out of 978
174 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
175 out of 978
176 out of 978
177 out of 978
178 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=captain&include_fields=email,real_nam/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: e63fd4fc05a0 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author Ms2ger@gmail.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 924690f9d81b is related to another bug (975277).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ba884d47b839 is related to another bug (966992).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision bbab21ac3b8c is related to another bug (932080).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author dholbert@mozilla.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 1a1a66376625 is related to another bug (1123507).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 1a4c6cb31743 is related to another bug (1035773).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 5a4bb2092618 is related to another bug (915482).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision bf930e7d61d3 is related to another bug (911786).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 47a8380519cc was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 8f4ec9046b03 is related to another bug (836875).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 57bbbda0dedd is related to another bug (532150).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c8a1314aa449 is related to another bug (821998).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 5a74a94f6d44 is related to another bug (817342).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 31268d71c33c was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 5cab33f5bfbd was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 57bbbda0dedd was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 00244ceddd42 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 00244ceddd42 is related to another bug (733614).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 906c7ac5ac40 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision cf37257f81d2 is related to another bug (1195162).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 368852aa96c5 is related to another bug (1033424).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author jwwang@mozilla.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
e')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 769048: 'email'
179 out of 978
180 out of 978
181 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
182 out of 978
183 out of 978
Error with bug 772679: Too many matching authors (Ms2ger@gmail.com, roc@ocallahan.org) found for ms2ger@gmail.com
184 out of 978
185 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
186 out of 978
187 out of 978
188 out of 978
189 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=captain&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 769048: 'email'
190 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
191 out of 978
192 out of 978
193 out of 978
194 out of 978
Error with bug 1135541: Too many matching authors (dholbert@mozilla.com, gsquelart@mozilla.com) found for dholbert@cs.stanford.edu
195 out of 978
196 out of 978
197 out of 978
198 out of 978
199 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
200 out of 978
201 out of 978
202 out of 978
203 out of 978
204 out of 978
205 out of 978
206 out of 978
207 out of 978
208 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
209 out of 978
210 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kamidphish&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1229585: 'email'
211 out of 978
212 out of 978
213 out of 978
214 out of 978
215 out of 978
216 out of 978
217 out of 978
Error with bug 1159171: Too many matching authors (jyavenard@mozilla.com, jwwang@mozilla.com) found for jwwang@mozilla.com
218 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
219 out of 978
220 out of 978/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 44ae8462d6ab looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author wkocher@mozilla.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision a179552e0d48 is related to another bug (501739).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b6dc96f18391 is related to another bug (853301).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Reviewer jimb@mozilla.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 3c4dc1b74b7f is related to another bug (1125776).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 600aee747c40 is related to another bug (892378).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 0e9b37fd47c6 is related to another bug (1125581).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Reviewer jmathies@mozilla.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
analyze_commits.py:81: UserWarning: Reviewer philipp@mozilla.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7ba8c86f1a56 is related to another bug (885591).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 76820c6dff7b is related to another bug (579517).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: a81349b61b4e looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision a81349b61b4e is related to another bug (886526).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision d4f1b39a8575 is related to another bug (898558).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author honzab.moz@firemni.cz is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b91deae84856 is related to another bug (1057914).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision efe5844c9a5b is related to another bug (1057879).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 0c8faa128dff looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c6962b015a6a is related to another bug (874721).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision introducting was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision intermittent was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 04d60f86b935 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision acc1c0494f49 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 78424c3ea6cb was not found.
  revs, backout_comments = get_commits_for_bug(bug)

221 out of 978
222 out of 978
223 out of 978
224 out of 978
225 out of 978
226 out of 978
Error with bug 1166924: Too many matching authors (amarchesini@mozilla.com, wkocher@mozilla.com) found for wkocher@mozilla.com
227 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
228 out of 978
229 out of 978
230 out of 978
231 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=lots&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 924839: 'email'
232 out of 978
233 out of 978
Error with bug 1160445: 'blocks'
234 out of 978
235 out of 978
236 out of 978
237 out of 978
238 out of 978
239 out of 978
240 out of 978
241 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=makoto&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 839031: 'email'
242 out of 978
243 out of 978
244 out of 978
245 out of 978
246 out of 978
247 out of 978
248 out of 978
249 out of 978
250 out of 978
251 out of 978
Error with bug 1242904: Uplift either accepted or rejected.
252 out of 978
253 out of 978
254 out of 978
255 out of 978
256 out of 978
257 out of 978
258 out of 978
259 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
260 out of 978
261 out of 978
262 out of 978
263 out of 978
264 out of 978
265 out of 978
266 out of 978
267 out of 978
268 out of 978
Error with bug 712363: Too many matching authors (honzab.moz@firemni.cz, brian@briansmith.org) found for honzab.moz@firemni.cz
269 out of 978
270 out of 978
271 out of 978
272 out of 978
273 out of 978
274 out of 978
275 out of 978
276 out of 978
277 out of 978
278 out of 978
Error with bug 1073805: Too many matching authors (cpearce@mozilla.com, giles@thaumas.net) found for cpearce@mozilla.com
279 out of 978
280 out of 978
Error with bug 1072780: Too many matching authors (dholbert@mozilla.com, rjesup@jesup.org) found for dholbert@cs.stanford.edu
281 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
282 out of 978
283 out of 978
284 out of 978
285 out of 978
286 out of 978
287 out of 978
288 out of 978
289 out of 978
290 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 813867: 'email'
291 out of 978
292 out of 978
293 out of 978
294 out of 978
295 out of 978
296 out of 978
297 out of 978
298 out of 978
299 out of 978
300 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=josh&include_fields=email,real_name')
('   text: ', u'{"documenta/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision bb3866dea03e is related to another bug (1113425).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7e9c4abf391b is related to another bug (1155823).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 962e15b81684 is related to another bug (947753).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author steve@ssinger.info is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1205083 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
analyze_commits.py:81: UserWarning: Author slee@mozilla.com could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f74f1e59dc50 is related to another bug (899050).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: d0801a6d38dc looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author jmathies@mozilla.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
analyze_commits.py:81: UserWarning: Author mycoolclub@yahoo.com could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 35bd769b49f8 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e4f3d4279b4c was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ca467297fa07 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b4851ce6637d was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 0c8faa128dff is related to another bug (1072780).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c8d6e8deac02 is related to another bug (936681).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b4a630050f61 is related to another bug (933322).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1154683 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 3bd19a1cf4b4 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author ajones@mozilla.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
analyze_commits.py:81: UserWarning: Reviewer gsquelart@mozilla.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
tion":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1135491: 'email'
301 out of 978
302 out of 978
303 out of 978
304 out of 978
305 out of 978
306 out of 978
Error with bug 1207378: Too many matching authors (seth.bugzilla@blackhail.net, steve@ssinger.info) found for steve@ssinger.info
307 out of 978
308 out of 978
309 out of 978
310 out of 978
311 out of 978
312 out of 978
313 out of 978
314 out of 978
315 out of 978
316 out of 978
317 out of 978
318 out of 978
319 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Steven%20Lee&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
320 out of 978
321 out of 978
322 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
323 out of 978
324 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 901211: 'email'
325 out of 978
326 out of 978
327 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=mycoolclub&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
328 out of 978
329 out of 978
330 out of 978
331 out of 978
332 out of 978
333 out of 978
334 out of 978
335 out of 978
336 out of 978
337 out of 978
338 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
339 out of 978
340 out of 978
Error with bug 1098126: Too many matching authors (karlt@mozbugz.karlt.net, ajones@mozilla.com) found for karlt+@karlt.net
341 out of 978
342 out of 978
343 out of 978
344 out of 978
345 out of 978
346 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 822148: 'email'
347 out of 978
348 out of 978
349 out of 978
350 out of 978
351 out of 978
352 out of 978
353 out of 978
354 out of 978
355 out of 978
356 out of 978
357 out of 978
Error with bug 1131638: Too many matching authors (matt.woodrow@gmail.com, ajones@mozilla.com) found for ajones@mozilla.com
358 out of 978
359 out of 978
360 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
361 out of 978
362 out of 978
363 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/us/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: a081cfffeed0 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision be3cef4b69e3 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 6eb4d1008076 is related to another bug (905150).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 904598 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 758363 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1151309 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ed488f577c84 is related to another bug (733553).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f3e4bb1f935c is related to another bug (1199049).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 3260a147595a was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision bd1eed74e2ec was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 87ed776cbacd is related to another bug (1014466).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e79927f90d0b is related to another bug (518659).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Reviewer josh@joshmatthews.net is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
analyze_commits.py:81: UserWarning: Reviewer terrence.d.cole@gmail.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
er?match=sewardj&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 683127: 'email'
364 out of 978
365 out of 978
366 out of 978
367 out of 978
368 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 822148: 'email'
369 out of 978
370 out of 978
371 out of 978
372 out of 978
373 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
374 out of 978
375 out of 978
376 out of 978
377 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kamidphish&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1171228: 'email'
378 out of 978
379 out of 978
380 out of 978
381 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 820652: 'email'
382 out of 978
383 out of 978
384 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 822700: 'email'
385 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 787899: 'email'
386 out of 978
387 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=mt&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1072044: 'email'
388 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=sicking&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1207556: 'email'
389 out of 978
390 out of 978
391 out of 978
392 out of 978
393 out of 978
394 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
395 out of 978
Connection err/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 1c1981ecaba8 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e7a44e5b762f is related to another bug (985623).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e10a3c75e704 is related to another bug (789096).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 679616f29acc is related to another bug (960648).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b887641e3983 is related to another bug (950240).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Reviewer matt.woodrow@gmail.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 3044539fec87 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Reviewer bzbarsky@mit.edu is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ccba2b6b092d is related to another bug (946479).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 921be552b22f is related to another bug (943482).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 33e4d9281b5d is related to another bug (947429).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 567d2bba4f97 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 733c4bc77ab5 is related to another bug (622936).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 0e50480c408f is related to another bug (609163).
  revs, backout_comments = get_commits_for_bug(bug)
or:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=bbouvier&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 999651: 'email'
396 out of 978
397 out of 978
398 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1073250: 'email'
399 out of 978
400 out of 978
401 out of 978
402 out of 978
403 out of 978
404 out of 978
405 out of 978
Error with bug 810470: Too many matching authors (roc@ocallahan.org, ryanvm@gmail.com) found for ryanvm@gmail.com
406 out of 978
407 out of 978
408 out of 978
409 out of 978
410 out of 978
411 out of 978
412 out of 978
413 out of 978
414 out of 978
415 out of 978
416 out of 978
417 out of 978
418 out of 978
419 out of 978
420 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
421 out of 978
422 out of 978
423 out of 978
424 out of 978
425 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 804303: 'email'
426 out of 978
427 out of 978
428 out of 978
429 out of 978
430 out of 978
431 out of 978
432 out of 978
433 out of 978
434 out of 978
435 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=mt&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1159641: 'email'
436 out of 978
437 out of 978
438 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 822148: 'email'
439 out of 978
440 out of 978
441 out of 978
442 out of 978
443 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=lots&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 924839: 'email'
444 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
445 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Jacek%20Caban&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any uanalyze_commits.py:81: UserWarning: Author jacek@codeweavers.com could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 711656 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision be4ba3d5ca9a is related to another bug (1085759).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 5dd87bff4e11 is related to another bug (203271).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 5cae5d9ef313 is related to another bug (957693).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 3c26af850666 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 603008 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 50edbeb8912e was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7f904a453bdb was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 6ddbdbd66174 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e17123b5bafd was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e98a176b4e0b was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision cf98685302b6 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c85c1d11bd72 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 307ab52a74fc is related to another bug (867632).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 00c70533490e is related to another bug (803414).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 377f51d0e354 is related to another bug (938108).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 377f51d0e354 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 1c6223f7c74f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision d1b5408d946f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 044aec58f5b3 is related to another bug (964929).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 974df2f038ae is related to another bug (959222).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 95c958fa3cf2 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author roc@ocallahan.org is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 820875 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 9400b5b4f6da is related to another bug (1186745).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 83024a3a9586 is related to another bug (1186745).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 91a3fb327007 is related to another bug (1159747).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 91a3fb327007 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c99d19bfd6a0 is related to another bug (1149605).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 387a5f8b5e4f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author wjohnston@mozilla.com could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
analyze_commits.py:81: UserWarning: Reviewer dholbert@mozilla.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
ser information."}')
446 out of 978
447 out of 978
448 out of 978
449 out of 978
450 out of 978
451 out of 978
452 out of 978
453 out of 978
454 out of 978
455 out of 978
456 out of 978
Error with bug 810470: Too many matching authors (roc@ocallahan.org, ryanvm@gmail.com) found for ryanvm@gmail.com
457 out of 978
458 out of 978
459 out of 978
460 out of 978
461 out of 978
462 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=bschouten&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1130195: 'email'
463 out of 978
464 out of 978
465 out of 978
466 out of 978
467 out of 978
468 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 775868: 'email'
469 out of 978
470 out of 978
Error with bug 1131638: Too many matching authors (matt.woodrow@gmail.com, ajones@mozilla.com) found for ajones@mozilla.com
471 out of 978
472 out of 978
473 out of 978
474 out of 978
475 out of 978
476 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=sicking&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1233098: 'email'
477 out of 978
478 out of 978
479 out of 978
480 out of 978
481 out of 978
482 out of 978
483 out of 978
484 out of 978
485 out of 978
486 out of 978
487 out of 978
488 out of 978
489 out of 978
Error with bug 870941: Too many matching authors (spohl.mozilla.bugs@gmail.com, roc@ocallahan.org) found for robert@ocallahan.org
490 out of 978
491 out of 978
492 out of 978
493 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
494 out of 978
495 out of 978
496 out of 978
497 out of 978
498 out of 978
499 out of 978
500 out of 978
501 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=tuexen&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 859179: 'email'
502 out of 978
503 out of 978
504 out of 978
505 out of 978
506 out of 978
507 out of 978
508 out of 978
509 out of 978
510 out of 978
511 out of 978
512 out of 978
513 out of 978
514 out of 978
515 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Wes%20Johnston&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Wes%20Johnston&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
516 out of 978
517 out of 978
518 out of 978
519 out of 978
520 out of 978
Connection error:
('   url: ', u'https://bugzilanalyze_commits.py:81: UserWarning: Author jib@mozilla.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 6cfd941c49a6 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 6cfd941c49a6 is related to another bug (776142).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision afd9e75374e8 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 0670cdaf7e9c is related to another bug (891400).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1151821 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 27cc305a5a1a is related to another bug (702739).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: d75c06f195d2 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author bugspam.Callek@gmail.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 877e91964ea9 is related to another bug (1108145).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 2390cb215cc4 is related to another bug (918825).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 3bc7ae253cdf is related to another bug (924596).
  revs, backout_comments = get_commits_for_bug(bug)
la.mozilla.org/rest/user?match=jrmuziel&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 934860: 'email'
521 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 717853: 'email'
522 out of 978
523 out of 978
524 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ryan%20VanderMeulen&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
525 out of 978
526 out of 978
527 out of 978
528 out of 978
529 out of 978
530 out of 978
531 out of 978
532 out of 978
533 out of 978
534 out of 978
535 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=bgirard&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 687267: 'email'
536 out of 978
537 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 980493: 'email'
538 out of 978
539 out of 978
540 out of 978
541 out of 978
542 out of 978
543 out of 978
544 out of 978
545 out of 978
546 out of 978
547 out of 978
548 out of 978
549 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 822148: 'email'
550 out of 978
551 out of 978
552 out of 978
Error with bug 771251: Too many matching authors (bugspam.Callek@gmail.com, benjamin@smedbergs.us) found for Callek@gmail.com
553 out of 978
554 out of 978
555 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 819817: 'email'
556 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
557 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=sicking&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 980243: 'email'
558 out of 978
559 out of 978
560 out of 978
561 out of 978
Error with bug 712363: Too many matching authors (honzab.moz@firemni.cz, brian@briansmith.org) found /home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 39f872a217ff looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author jacek@codeweavers.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1133119 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 68551387765a is related to another bug (1042689).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1048133 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision be0d3f1de9c2 is related to another bug (807181).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 01123067aa58 is related to another bug (806256).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 821358 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
analyze_commits.py:81: UserWarning: Author doug.turner@gmail.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 86a3edf2960d was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 1a8b11ae0f4b was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e02a02fd1208 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 3aa6bab70380 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: fe7f450a5435 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 86f9003c1251 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 9989f9ef7b90 is related to another bug (10209).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1119019 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 840480 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 9e126e97e347 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
for honzab.moz@firemni.cz
562 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
563 out of 978
564 out of 978
565 out of 978
566 out of 978
567 out of 978
568 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=sicking&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 690418: 'email'
569 out of 978
570 out of 978
571 out of 978
572 out of 978
573 out of 978
Error with bug 1258317: Too many matching authors (jacek@codeweavers.com, cyu@mozilla.com) found for jacek@codeweavers.com
574 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=sewardj&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 683127: 'email'
575 out of 978
Error with bug 712363: Too many matching authors (honzab.moz@firemni.cz, brian@briansmith.org) found for honzab.moz@firemni.cz
576 out of 978
577 out of 978
578 out of 978
579 out of 978
580 out of 978
581 out of 978
582 out of 978
583 out of 978
584 out of 978
585 out of 978
586 out of 978
587 out of 978
588 out of 978
589 out of 978
590 out of 978
591 out of 978
592 out of 978
593 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=mt&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1133866: 'email'
594 out of 978
595 out of 978
596 out of 978
597 out of 978
598 out of 978
599 out of 978
600 out of 978
601 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
602 out of 978
603 out of 978
604 out of 978
Error with bug 1156063: Too many matching authors (doug.turner@gmail.com, bent.mozilla@gmail.com) found for doug.turner@gmail.com
605 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=sicking&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1205456: 'email'
606 out of 978
607 out of 978
608 out of 978
609 out of 978
610 out of 978
611 out of 978
612 out of 978
613 out of 978
614 out of 978
615 out of 978
616 out of 978
617 out of 978
618 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kamidphish&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1105535: 'email'
619 out of 978
620 out of 978
621 out of 978
622 out of 978
623 out of 978
624 out of 978
625 out of 978
626 out of 978
627 out of 978
628 out of 978
629 out of 978
630 out of 978
631 out of 978
632 out of 978
633 out of 978
634 out of 978
635 out of 978
636 out of 978
637 ou/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 9f476b4ac1e1 is related to another bug (762966).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 6b58397ad735 is related to another bug (795262).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b274e8e3479f is related to another bug (794694).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f23efef7f616 is related to another bug (660156).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author mozilla@noorenberghe.ca could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 695049af7654 is related to another bug (999580).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 1bc5b733beac was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ec145c86be71 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 477094180dff was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c0b8aef8b570 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1085175 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ec3acc5237a8 is related to another bug (1181763).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 5b6d7523e355 is related to another bug (804174).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author jseward@acm.org is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
t of 978
638 out of 978
639 out of 978
640 out of 978
641 out of 978
642 out of 978
643 out of 978
644 out of 978
645 out of 978
646 out of 978
647 out of 978
648 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
649 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Matthew%20Noorenberghe&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
650 out of 978
651 out of 978
Error with bug 1240589: Too many matching authors (jacek@codeweavers.com, cpeterson@mozilla.com) found for jacek@codeweavers.com
652 out of 978
653 out of 978
654 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=mt&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1037424: 'email'
655 out of 978
656 out of 978
657 out of 978
658 out of 978
659 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
660 out of 978
661 out of 978
Error with bug 1072780: Too many matching authors (dholbert@mozilla.com, rjesup@jesup.org) found for dholbert@cs.stanford.edu
662 out of 978
663 out of 978
664 out of 978
665 out of 978
666 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=josh&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1123245: 'email'
667 out of 978
668 out of 978
669 out of 978
670 out of 978
671 out of 978
672 out of 978
673 out of 978
674 out of 978
675 out of 978
676 out of 978
677 out of 978
678 out of 978
679 out of 978
680 out of 978
681 out of 978
682 out of 978
683 out of 978
684 out of 978
685 out of 978
686 out of 978
687 out of 978
688 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=sewardj&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 683127: 'email'
689 out of 978
690 out of 978
691 out of 978
692 out of 978
693 out of 978
694 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 967844: 'email'
695 out of 978
696 out of 978
697 out of 978
698 out of 978
699 out of 978
700 out of 978
701 out of 978
702 out of 978
703 out of 978
704 out of 978
705 out of 978
706 out of 978
707 out of 978
708 out of 978
709 out of 978
710 out of 978
711 out of 978
712 out of 978
713 out of 978
714 out of 978
715 out of 978
716 out of 978
717 out of 978
718 out of 978
719 out of 978
720 out of 978
721 out of 978
722 out of 978
723 out of 978
724 out o/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ce4011f33422 is related to another bug (933882).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c6981912ff87 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision fe63bf2c6a3a was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b8d3be6e3c1a was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 555e5759fe5f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 5e6899ab5ead was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 82495f0c5da2 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 9f3212effb9f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ce4011f33422 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 77be849d81e7 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ae6f2151610f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 2abeb02c4777 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author shu@rfrn.org could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b40ba117fa75 is related to another bug (1079858).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1198435 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b4ebefd0f7e3 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Reviewer bas@basschouten.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 0b7c27024048 is related to another bug (854323).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 03682dcd82bb looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 71da07ec8ce3 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 698882 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 9f11fc87eaa6 is related to another bug (679472).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f097100df8aa is related to another bug (549468).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision cc1e08803869 is related to another bug (672054).
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author sphink@gmail.com is not in the list of authors on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
analyze_commits.py:81: UserWarning: Reviewer jmuizelaar@mozilla.com is not in the list of reviewers on Bugzilla.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 76fb1757892d looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f309dacf4010 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
f 978
725 out of 978
726 out of 978
727 out of 978
728 out of 978
729 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 839117: 'email'
730 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Shu-yu%20Guo&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
731 out of 978
732 out of 978
733 out of 978
734 out of 978
735 out of 978
736 out of 978
737 out of 978
738 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1257369: 'email'
739 out of 978
740 out of 978
741 out of 978
742 out of 978
743 out of 978
744 out of 978
745 out of 978
746 out of 978
747 out of 978
748 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
749 out of 978
Error with bug 1126465: Too many matching authors (bobbyholley@gmail.com, ryanvm@gmail.com) found for ryanvm@gmail.com
750 out of 978
751 out of 978
752 out of 978
753 out of 978
754 out of 978
755 out of 978
756 out of 978
757 out of 978
758 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=josh&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 802355: 'email'
759 out of 978
760 out of 978
761 out of 978
762 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=closed&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1028358: 'email'
763 out of 978
764 out of 978
765 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
766 out of 978
767 out of 978
768 out of 978
769 out of 978
770 out of 978
771 out of 978
772 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=unfocussed&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1124400: 'email'
773 out of 978
774 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Daniel%20Holbert&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to acanalyze_commits.py:81: UserWarning: Author dholbert@cs.stanford.edu could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision dab8e3865967 is related to another bug (789096).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1044975 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision fe0abb4decb6 is related to another bug (819731).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 0319dd845d53 is related to another bug (815666).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c6970a82a686 is related to another bug (819264).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e6f113c83095 is related to another bug (741295).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7d8224f00aec is related to another bug (1137336).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f88b64c38d3d was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1132572 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 3ead3466f84a is related to another bug (1172264).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision dfdcce6b319a is related to another bug (748550).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 681ab70f6707 is related to another bug (949334).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 093a4a3a68ca is related to another bug (913734).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision a145ded68994 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 3127df59e0dc is related to another bug (1110871).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision be09855c0f5c is related to another bug (626245).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1097437 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 865919 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
cess any user information."}')
775 out of 978
776 out of 978
777 out of 978
778 out of 978
779 out of 978
780 out of 978
781 out of 978
782 out of 978
783 out of 978
784 out of 978
785 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jdaggett&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1074223: 'email'
786 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jrmuziel&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 934860: 'email'
787 out of 978
Error with bug 1126465: Too many matching authors (bobbyholley@gmail.com, ryanvm@gmail.com) found for ryanvm@gmail.com
788 out of 978
789 out of 978
790 out of 978
791 out of 978
792 out of 978
793 out of 978
794 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 967844: 'email'
795 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
796 out of 978
797 out of 978
798 out of 978
799 out of 978
800 out of 978
801 out of 978
802 out of 978
803 out of 978
804 out of 978
805 out of 978
806 out of 978
807 out of 978
808 out of 978
809 out of 978
810 out of 978
811 out of 978
812 out of 978
813 out of 978
814 out of 978
815 out of 978
816 out of 978
817 out of 978
818 out of 978
Error with bug 1126465: Too many matching authors (bobbyholley@gmail.com, ryanvm@gmail.com) found for ryanvm@gmail.com
819 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=bgirard&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 804606: 'email'
820 out of 978
821 out of 978
822 out of 978
823 out of 978
824 out of 978
825 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
826 out of 978
827 out of 978
828 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
829 out of 978
830 out of 978
831 out of 978
832 out of 978
833 out of 978
834 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 822148: 'email'
835 out of 978
Connecanalyze_commits.py:81: UserWarning: Author chris.double@double.co.nz could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c9455df0575c is related to another bug (884310).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c76e6b6d4591 is related to another bug (1149420).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 39db16b78175 is related to another bug (590568).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ec3e4786877d is related to another bug (621347).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7ccb164032a0 is related to another bug (644747).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 3785489fa210 is related to another bug (1222405).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 06ea22b8654c is related to another bug (783533).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 1f6cd8529ae9 is related to another bug (783449).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 360ab7771e27 is related to another bug (783856).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f9a8fdb08193 is related to another bug (784255).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1135408 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 71a73815eab4 is related to another bug (418131).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision c52ed5e53db9 is related to another bug (621551).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision a16971b9a582 is related to another bug (692109).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 57cf4086191d is related to another bug (731503).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision d0477ba39a89 is related to another bug (720846).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7672adec56b9 is related to another bug (730581).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 2067107a71ca looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision f98f80d2126c is related to another bug (915763).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7a1c696cade6 is related to another bug (974201).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 7d17f2d129e3 is related to another bug (881959).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 58dbd84ae828 is related to another bug (917246).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision fb89f2090779 is related to another bug (881959).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision fe576415129e is related to another bug (881959).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 5e5d5e42f6c2 is related to another bug (881959).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b221626331b4 is related to another bug (881959).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 2aa38a1a2b06 is related to another bug (881959).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 3d2aac8dc618 is related to another bug (916773).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ade49a801461 is related to another bug (881959).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 4323508b3412 is related to another bug (917260).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision dedaffb8297a is related to another bug (1046055).
  revs, backout_comments = get_commits_for_bug(bug)
tion error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 822148: 'email'
836 out of 978
837 out of 978
838 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=josh&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 810494: 'email'
839 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Chris%20Double&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
840 out of 978
841 out of 978
842 out of 978
843 out of 978
844 out of 978
845 out of 978
846 out of 978
847 out of 978
848 out of 978
849 out of 978
850 out of 978
851 out of 978
852 out of 978
853 out of 978
854 out of 978
855 out of 978
856 out of 978
857 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 980493: 'email'
858 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
859 out of 978
860 out of 978
861 out of 978
862 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=matspal&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 626245: 'email'
863 out of 978
864 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 819772: 'email'
865 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
866 out of 978
867 out of 978
868 out of 978
869 out of 978
870 out of 978
871 out of 978
872 out of 978
873 out of 978
874 out of 978
875 out of 978
876 out of 978
877 out of 978
878 out of 978
879 out of 978
880 out of 978
881 out of 978
882 out of 978
883 out of 978
884 out of 978
885 out of 978
886 out of 978
887 out of 978
888 out of 978
889 out of 978
890 out of 978
891 out of 978
892 out of 978
893 out of 978
894 out of 978
895 out of 978
896 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" ar/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: a10dd1a539db looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision d9ad4cc32e5d was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision e13e98eab890 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 011dcedf181f was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision fb230c191a88 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision caddb604d934 was not found.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision a2a645bf7ccf was not found.
  revs, backout_comments = get_commits_for_bug(bug)
analyze_commits.py:81: UserWarning: Author ryan could not be found.
  info = patchanalysis.bug_analysis(bug if bug is not None else bug_id, channel)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b077059c575a is related to another bug (724283).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision b61b7f2e0d0b is related to another bug (888562).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision d42a2a82f3d2 is related to another bug (789919).
  revs, backout_comments = get_commits_for_bug(bug)
gument to this function to access any user information."}')
897 out of 978
898 out of 978
899 out of 978
900 out of 978
901 out of 978
902 out of 978
903 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 820652: 'email'
904 out of 978
905 out of 978
906 out of 978
907 out of 978
908 out of 978
909 out of 978
910 out of 978
911 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1002426: 'email'
912 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 982141: 'email'
913 out of 978
914 out of 978
915 out of 978
916 out of 978
917 out of 978
918 out of 978
919 out of 978
920 out of 978
921 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=%3Cryan&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
922 out of 978
923 out of 978
924 out of 978
925 out of 978
926 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1072645: 'email'
927 out of 978
928 out of 978
929 out of 978
930 out of 978
931 out of 978
932 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
933 out of 978
934 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
935 out of 978
936 out of 978
937 out of 978
938 out of 978
939 out of 978
940 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=enndeakin&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 750061: 'email'
941 out of 978
942 out of 978
943 out of 978
944 out of 978
945 out of 978
946 out of 978
947 out of 978
Error with bug 1126465: Too many matching authors (bobbyholley@gmail.com, ryanvm@gmail.com) found for ryanvm@gmail.com
948 out of 978
949 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":5/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 4bee0517d440 is related to another bug (568593).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 126cfa64a877 is related to another bug (568593).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: 82c3097bb8b3 looks like a backout, but we couldn't find which revision was backed out.
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 568953 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 824009fbc42f is related to another bug (1183967).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision 26f83c4cb81e is related to another bug (858914).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision ebdf97f9ec52 is related to another bug (866675).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:406: UserWarning: Revision a43be719866e is related to another bug (915495).
  revs, backout_comments = get_commits_for_bug(bug)
/home2/marco.castelluccio/Documenti/workspace/uplift-analysis/venv/local/lib/python2.7/site-packages/libmozdata/patchanalysis.py:481: UserWarning: Bug 1138391 doesn't have a uplift request date.
  info.update(uplift_info(bug, uplift_channel))
05,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
950 out of 978
951 out of 978
952 out of 978
953 out of 978
954 out of 978
955 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 804303: 'email'
956 out of 978
957 out of 978
958 out of 978
959 out of 978
960 out of 978
961 out of 978
962 out of 978
963 out of 978
964 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
965 out of 978
966 out of 978
967 out of 978
968 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
969 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=kats&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 1002426: 'email'
970 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=jlebar&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
Error with bug 822148: 'email'
971 out of 978
Error with bug 810470: Too many matching authors (roc@ocallahan.org, ryanvm@gmail.com) found for ryanvm@gmail.com
972 out of 978
973 out of 978
974 out of 978
975 out of 978
976 out of 978
Connection error:
('   url: ', u'https://bugzilla.mozilla.org/rest/user?match=Ehsan%20Akhgari&include_fields=email,real_name')
('   text: ', u'{"documentation":"http://www.bugzilla.org/docs/tip/en/html/api/","error":true,"code":505,"message":"Logged-out users cannot use the \\"match\\" argument to this function to access any user information."}')
977 out of 978
978 out of 978
